src/framework/xr/xr-manager.js

```diff
@@ -581,8 +581,8 @@ class XrManager extends EventHandler {
             if (this._camera) {
                 this._camera.off('set_nearClip', onClipPlanesChange);
                 this._camera.off('set_farClip', onClipPlanesChange);
-
                 this._camera.camera.xr = null;
```

Comment:
we should add the `xr` property on a camera, it currently does not exist. (existing issue, I know)

Reply:
in its setter, it should also handle setting `_projMatDirty` so we don't need to poke it in like this